.github/actions/setup-cli-deps/action.yml

```diff
@@ -13,7 +13,7 @@ runs:
         git config --global user.name "App Management"
       shell: bash
     - name: Install pnpm
-      uses: pnpm/action-setup@c3b53f6a16e57305370b4ae5a540c2077a1d50dd # pin@v2.2.4
+      uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # pin@v4.0.0
```

Comment:
Verified this commit hash corresponds with the [authentic repo and release](https://github.com/pnpm/action-setup/releases/tag/v4.0.0).